pkg/api/api.go

```diff
@@ -71,6 +71,10 @@ type Config struct {
 	// initialized, the custom config handler will be used instead of
 	// DefaultConfigHandler.
 	CustomConfigHandler ConfigHandler `yaml:"-"`
+
+	// The values from the flags that have been set.
+	// Used for `/api/v1/status/flags` API.
+	Flags map[string]string `yaml:"-"`
```

Comment:
I'm not sure does this place is a good place to add it. If you have any suggestions, please help me to find a better place! Thanks!